composer/algorithms/factorize/factorize.py

```diff
@@ -28,16 +28,16 @@ def apply_factorization(model: torch.nn.Module,
                         min_features: int = 512,
                         latent_features: Union[int, float] = 0.25,
                         optimizers: Optional[Union[Optimizer, Sequence[Optimizer]]] = None) -> torch.nn.Module:
-    """Replaces :class:`~torch.nn.Linear` and :class:`~torch.nn.Conv2d` modules and with
+    """Replaces :class:`~torch.nn.Linear` and :class:`~torch.nn.Conv2d` modules with
     :class:`~composer.algorithms.factorize.FactorizedLinear` and
     :class:`~composer.algorithms.factorize.FactorizedConv2d` modules.
 
-    Factorized modules replace one full-rank operation with a sequence of two
+    Factorized modules replace one, full-rank operation with a sequence of two,
```

Comment:
extra commas look weird to me; think these are cumulative adjectives that can't be reordered, and therefore don't get a comma: https://blogs.stockton.edu/helpwithpunctuation/files/2011/01/commas-with-adjectives.pdf

Reply:
You're right, my bad.